react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+class PokemonList extends React.Component {
+
+  render() {
+    const pokemonList = pokedex.map(pokemon =>
+      <li key={pokedex.toString()}>{pokemon.name}</li>
```

Comment:
Here the `key` should use the `number` property. The `key` should be unique for each item and is usually an `id` or something similar.
```JSX
<li key={pokemon.number>{pokemon.name}</li>
```